e2e-tests/tfm/cypress/e2e/journeys/case-amendments/user-flow/1. add-amendment-request.spec.js

```diff
@@ -112,36 +108,28 @@ context('Amendments - Request date', () => {
     facilityPage.facilityTabAmendments().click();
     amendmentsPage.addAmendmentButton().click();
 
-    cy.keyboardInput(amendmentsPage.amendmentRequestDayInput(), dateConstants.todayDay);
-    cy.keyboardInput(amendmentsPage.amendmentRequestMonthInput(), dateConstants.todayMonth);
-    cy.keyboardInput(amendmentsPage.amendmentRequestYearInput(), '22');
+    cy.completeDateFormFields({ idPrefix: 'amendment--request-date', year: '22' });
 
     cy.clickContinueButton();
 
     errorSummary().contains('The year for the amendment request date must include 4 numbers');
     amendmentsPage.errorMessage().contains('The year for the amendment request date must include 4 numbers');
 
-    cy.keyboardInput(amendmentsPage.amendmentRequestDayInput(), dateConstants.todayDay);
-    cy.keyboardInput(amendmentsPage.amendmentRequestMonthInput(), dateConstants.todayMonth);
-    cy.keyboardInput(amendmentsPage.amendmentRequestYearInput(), '2O22');
+    cy.completeDateFormFields({ idPrefix: 'amendment--request-date', year: '2022' });
```

Comment:
Another classic 2o22 here! (might be worth searching the codebase for `year: '2022' });` and checking all the cases